libs/tasks/interface-adapters/src/lib/dto/user-task.dto.ts

```
@@ -0,0 +1,46 @@
+export class UserTaskDto {
+  @Field(() => ID)
+  id: string;
+
+  @Field(() => Date)
+  createdAt: Date;
+
+  @Field(() => Date, { nullable: true })
+  updatedAt: Date | null;
+
+  @Field(() => String)
+  taskId: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)